akka-projection-jdbc/src/test/scala/akka/projection/jdbc/JdbcProjectionSpec.scala

```diff
@@ -248,7 +248,9 @@ class JdbcProjectionSpec
 
   private val concatHandlerFail4Msg = "fail on fourth envelope"
   private def findById(entityId: String) = {
-    withRepo(_.findById(entityId)).futureValue.get
+    eventually {
+      withRepo(_.findById(entityId)).futureValue.get
+    }
```

Comment:
Surprisingly enough, I've seen this code fail too. If this runs too early, the `futureValue` completes as a `None` causing the `.value` to fail.